server/services/src/main/java/org/zanata/rest/service/ResourceUtils.java

```diff
@@ -1392,13 +1398,17 @@ public boolean transferToTranslationsResource(TranslationsResource transRes,
         boolean found = this.transferToTranslationsResourceExtensions(document,
                 transRes.getExtensions(true), enabledExtensions, locale,
                 hTargets);
+        boolean showEmail = config.isDisplayUserEmail() || identity.hasRole("admin");
```

Comment:
Admin should not be able to see it either

Reply:
As discussed with @aeng, admins may need extra information like that to do their admin-type things. The main thing is to avoid showing the email address to the public.